### PR TITLE
Bump to Xcode 12.2 beta 1.

### PR DESCRIPTION
**Pull request by rolfbjarne**

### CODE REVIEW

**Comment by dalexsoto** [comment]
Thanls @rolfbjarne not sure what happened locally heh, I’ll take over and fix the tests etc

**Comment by dalexsoto** [comment]
Looks lime watchOS is having a hard time heh

**Review by tj-devel709 — APPROVED** [review]

File: src/gamecontroller.cs [hunk 1]
```diff
@@ -50,11 +50,11 @@ partial interface GCControllerElement {
 		[NullAllowed, Export ("localizedName", ArgumentSemantic.Strong)]
 		string LocalizedName { get; set; }
 
-		[NoTV, NoMac, iOS (14, 0)]
+		[TV (14, 0), Mac (11, 0), iOS (14, 0)]
```
Comment (rolfbjarne) [inline src/gamecontroller.cs:53]: Shouldn't this be tvOS 14.2?

Comment (dalexsoto) [reply src/gamecontroller.cs:53]: Whoops yup you are right

**Review by mandel-macaque — APPROVED** [review]

**Comment by dalexsoto** [comment]
build

File: Make.versions [hunk 2]
```diff
@@ -43,5 +43,5 @@
 #   line changed in git).
 #
 
-IOS_PACKAGE_VERSION=14.0.0.$(IOS_COMMIT_DISTANCE)
-MAC_PACKAGE_VERSION=6.99.0.$(MAC_COMMIT_DISTANCE)
+IOS_PACKAGE_VERSION=14.0.1.$(IOS_COMMIT_DISTANCE)
```
Comment (spouliot) [inline Make.versions:46]: too close
that should be `14.3` so we can release a stable `14.2` whenever (or if) Xcode 12.1 comes out

**Review by spouliot — CHANGES_REQUESTED** [review]

File: tests/common/TestRuntime.cs [hunk 3]
```diff
@@ -217,6 +217,18 @@ public static bool CheckXcodeVersion (int major, int minor, int build = 0)
...
+#endif
+			case 2:
+#if __WATCHOS__
+				return CheckWatchOSSystemVersion (7, 1);
+#elif __TVOS__
+				return ChecktvOSSystemVersion (14, 2);
+#elif __IOS__
+				return CheckiOSSystemVersion (14, 2);
+#elif MONOMAC
+				return CheckMacSystemVersion (11, 0, 0);
```
Comment (spouliot) [inline tests/common/TestRuntime.cs:229]: the 12,0 check will need to be brought back to 10,15 too

**Review by spouliot — APPROVED** [review]

**Pull request merged**